lib/browser/api/menu-item.js

```diff
@@ -7,18 +7,8 @@ let nextCommandId = 0
 const MenuItem = function (options) {
   const {Menu} = require('electron')
 
-  this.selector = options.selector
-  this.type = options.type
-  this.role = options.role
-  this.label = options.label
-  this.sublabel = options.sublabel
-  this.accelerator = options.accelerator
-  this.icon = options.icon
-  this.enabled = options.enabled
-  this.visible = options.visible
-  this.checked = options.checked
-
-  this.submenu = options.submenu
+  Object.assign(this, options)
```

Comment:
> and just before calling .assign, delete the properties from options which are in that array, aka

You don't want to mutate an object the user provided, it will most likely annoy them 😆 

Just set `options` to `Object.assign({}, options)` before doing this